ref/license-information.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud is built using multiple open source software components. This guide lists those components and where to find their license information.
+
+| Software components | Where to find their licenses |
+|--|--|
+| LXD images | Within the LXD image, `/usr/share/licenses` <br/> The copyright information of Anbox Cloud is available at `/usr/share/doc/*/copyright`. |
+| Android | In the Android container, <br/> **Settings** > **About emulated device** > **Legal information** |
+| Snaps | On the host system, <br/>`/snap/<SNAP_NAME>/current/` </br>Replace <SNAP_NAME> with the name of the Snap.|
```

Comment:
Not necessarily on the host system. As for appliance the snaps are installed within LXD containers as that is what we target Juju to.

```suggestion
| Snaps | <br/>`/snap/<SNAP_NAME>/current/` </br>Replace <SNAP_NAME> with the name of the Snap.|
```